packages/kolibri/components/CoreTable.vue

```diff
@@ -144,6 +144,14 @@
     text-align: left;
   }
 
+  /deep/ .core-table {
+    display: table;
+    width: 100%;
+    overflow-x: auto;
+    white-space: nowrap;
+    table-layout: fixed;
+  }
+
```

Comment:
Just above here there is a `.core-table` definition without `/deep/` - is there a need for the `/deep/` here?